sigma/backends/loki/loki.py

```diff
@@ -180,6 +182,43 @@ def finalize_expression(self) -> str:
         return f"{self.op} {or_value}"
 
 
+@dataclass
+class LogQLDeferredFieldRefExpression(DeferredQueryExpression):
+    """'Defer' field reference matching to pipelined command **BEFORE** main search expression."""
+
+    field: str
+    value: str
+    field_tracker: int
+
+    def finalize_expression(self) -> str:
+        return (
+            "match_"
+            + str(self.field_tracker)

```

Comment:
It feels to me like this could be achieved more effectively with an [f-string](https://docs.python.org/3/tutorial/inputoutput.html#tut-f-strings)

Reply:
Yes, addressed in 8aed4ef